app/javascript/croppable/index.js

```diff
@@ -0,0 +1,162 @@
+import Cropper from 'cropperjs';
```

Comment:
Why is this file duplicated? We have`/javascripts/croppable.esm.js` and `app/javascript/croppable/index.js`. Can we just keep one?